api/src/main/java/com/codeforcommunity/api/IProtectedSiteProcessor.java

```diff
@@ -1,10 +1,7 @@
 package com.codeforcommunity.api;
 
 import com.codeforcommunity.auth.JWTData;
-import com.codeforcommunity.dto.site.AddSiteRequest;
-import com.codeforcommunity.dto.site.AdoptedSitesResponse;
-import com.codeforcommunity.dto.site.RecordStewardshipRequest;
-import com.codeforcommunity.dto.site.UpdateSiteRequest;
+import com.codeforcommunity.dto.site.*;
```

Comment:
Try to avoid * for imports whenever possible. We want to explicitly write out each import that we use.

Reply:
Oh that's weird. I definitely didn't make this change but I'll fix it, thank you for pointing it out!